c/transform.c

```diff
@@ -94,35 +95,22 @@ void gcj2wgs(double gcjLat, double gcjLng, double *wgsLat, double *wgsLng) {
 }
 
 void gcj2wgs_exact(double gcjLat, double gcjLng, double *wgsLat, double *wgsLng) {
-	double initDelta = 0.01;
-	double threshold = 0.000001;
-	double dLat, dLng, mLat, mLng, pLat, pLng;
-	dLat = dLng = initDelta;
-	mLat = gcjLat - dLat;
-	mLng = gcjLng - dLng;
-	pLat = gcjLat + dLat;
-	pLng = gcjLng + dLng;
+	double dLat, dLng;
+	// n_iter=2: centimeter precision, n_iter=5: double precision
+	const int n_iter = 2;
```

Comment:
If your going to drop the threshold check I suggest 3 or 4 iterations not just 2 as my tests showed that to get to the right precision most times, I iterated 3 times.